R/create_hexsticker.R

```diff
@@ -0,0 +1,29 @@
+#' @title Make hexagonal logo for package
+#'
+#' @description This function makes a hexsticker in INBO style for the provided
+#' package name.
+#'
+#' @param package_name package name that should be mentioned on the hexsticker
+#' @param filename filename to save the sticker
+#'
+#' @return A figure is saved in the working directory or provided path
+#'
+#' @export
+#'
+#' @importFrom hexSticker sticker
+#'
+#' @family utils
+#'
+#' @examples
+#' create_hexsticker("checklist", filename = "hexsticker.png")
+
+create_hexsticker <- function(package_name, filename = "hexsticker.png") {
+  background <- system.file("inbo-empty.png", package = "checklist")
+  sticker(
```

Comment:
You'll have to add something like `requireNamespace("hexstickers")` when you have it in `Suggests:` rather than `Imports:`. Note that you'll need to test if the package is available.

Reply:
If not available, do I just drop a warning?

And also with respect to testing: maybe I better also test the input (package_name and filename)?